src/classic/clvm_tools/stages/stage_2/operators.rs

```diff
@@ -52,6 +52,36 @@ pub struct CompilerOperatorsInternal {
     opt_memo: RefCell<HashMap<AllocatorRefOrTreeHash, NodePtr>>,
 }
 
+/// Given a list of search paths, find a full path to a file whose partial name
+/// is given.  If the file can't be found in any search path, use the expression
+/// the user gave to cause the file to be searched for in the error result.
```

Comment:
Suggestion: Comment should state what happens if multiple paths match

Reply:
ok